.pre-commit-config.yaml

```diff
@@ -8,19 +8,14 @@ repos:
       - id: mixed-line-ending
         args: [--fix=lf]
       - id: debug-statements
-  # code formatting
-  - repo: https://github.com/PyCQA/flake8
-    rev: 6.1.0
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: "v0.1.1"
     hooks:
-      - id: flake8
-        args: [ --max-line-length=120 ]
+      - id: ruff
+        args: ["--fix"]
+        exclude: "tests/input/"
   - repo: https://github.com/psf/black
     rev: 23.10.1
     hooks:
       - id: black
         args: [--safe, --line-length=120]
-  - repo: https://github.com/PyCQA/isort
```

Comment:
Do ruff + black play well together for sorting imports? I remember that isort came after black due to them conflicting, however that was some time ago and likely fixed.

Reply:
Yeah ruff went out after black so it supports it without config. (there's also ruffmt now, but I like black for having only one option).